.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
Kindly add the  `test.md` file to the .gitignore. Its part of the project requirements
<img width="1392" alt="Screenshot 2022-10-26 at 11 37 32" src="https://user-images.githubusercontent.com/48546992/197977577-81977d61-e35d-4a64-8444-99cd4c78afa4.png">